spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe 'posts#index', type: :feature do
```

Comment:
- [ ] Hello, kindly make sure to add all test included in the project requirement and remove test cases that are not need

![image](https://github.com/BilalLiaquat7/blogapp/assets/112261237/c1b318de-fc63-4022-829b-9e55b9919472)